package.json

```diff
@@ -32,18 +32,19 @@
     "package-tarball": "npm run build && npm pack"
   },
   "devDependencies": {
-    "@sketch-hq/sketch-assistant-utils": "6.0.0",
+    "@sketch-hq/sketch-assistant-utils": "^6.4.0",
     "@types/jest": "26.0.4",
     "@types/node": "14.0.22",
-    "jest": "26.6.3",
+    "jest": "^27.0.3",
     "node-fetch": "2.6.1",
     "prettier": "2.0.5",
-    "ts-jest": "26.1.2",
-    "typescript": "3.9.6",
+    "ts-jest": "^27.0.2",
+    "typescript": "^4.3.2",
     "webpack": "4.43.0",
     "webpack-cli": "3.3.12"
   },
   "dependencies": {
-    "@sketch-hq/sketch-assistant-types": "6.0.0"
+    "@sketch-hq/sketch-assistant-types": "^6.4.0",
```

Comment:
To follow the tradition of pinned version, let's drop the `^` everywhere

Reply:
Okidoki